robinklaassen/aas2
# Feature/updater

Updater!

How to test:
- pull branch
- set .env vars
  - `UPDATER_GIT_SECRET=secret`
  - `UPDATER_GIT_BRANCH=feature/updater`
- run `artisan config:clear` and `artisan optimize:clear`
- checkout to previous commit (dont worry, it is just a newline removal) `git checkout HEAD^`
- goto `http://aas2.local/updater/version` see 61dc0ba
- goto `http://aas2.local/updater/update?secret=secret` wait and see output with b74f9c9
- goto `http://aas2.local/updater/version` see b74f9c9
